c2corg_api/models/cache_version.py

```diff
@@ -455,7 +461,8 @@ def get_cache_key(document_id, lang, document_type=None):
         first()
 
     return _format_cache_key(
-        document_id, lang, version[0] if version else None, document_type)
+        document_id, lang, version[0] if version else None, document_type,
+        custom_cache_key if custom_cache_key else None)
```

Comment:
Instead of `custom_cache_key if custom_cache_key else None`, you can simply use `custom_cache_key`.